crates/dojo-defi/src/constant_product_market/components.cairo

```diff
@@ -106,8 +106,10 @@ impl MarketImpl of MarketTrait {
 
         // dy = Y * dx / X
         let quantity_optimal = (reserve_quantity * amount) / reserve_amount;
+
         // Unscale and convert to usize
-        (quantity_optimal.into().try_into().unwrap() / ONE_u128).into().try_into().unwrap()
+        // Need intermediate conversion to felt252 because there's no impl for Into<FixedType, u128>
+        (quantity_optimal.into().try_into().unwrap() / ONE_u128).try_into().unwrap()
```

Comment:
Lets implement this conversion

Reply:
Created the pr here https://github.com/influenceth/cubit/pull/7

Reply:
This has been merged on main 👍 I've removed the intermediate conversions